gulpfile.babel.js

```diff
@@ -7,15 +7,41 @@ const autoprefix = require('gulp-autoprefixer');
 const minifyCSS = require('gulp-minify-css');
 const typescript = require('gulp-typescript');
 const browserSync = require('browser-sync');
-const del = require('del');
 const less = require('gulp-less');
 const gulpImport = require('gulp-html-import');
 const ejs = require('gulp-ejs');
 const plumber = require('gulp-plumber');
 const replace = require('gulp-replace');
-const connect = require('gulp-connect');
+const del = require('del');
+const fs = require('fs');
 const version = process.env.npm_package_version || '0.0.0';
 
+let config = require('./config.js') || {};
```

Comment:
`npm start` crashes with `code: 'MODULE_NOT_FOUND'` if `./config.js` doesn't exist. Probably need to wrap this in a try/catch instead.

Reply:
Good catch! I remember thinking I need to test this actually works without a config file, as require would presume it would throw if it doesn’t exist.
Will update it soon

Reply:
Update with a try catch, and a message if the file doesn't exist.